core/cache/backendTestCase_test.go

```diff
@@ -0,0 +1,167 @@
+package cache
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type BackendTestEntry struct {
+	Content string
+}
+
+type (
+	BackendTestCase struct {
+		T       *testing.T
+		Backend Backend
+	}
+)
+
+func (tc *BackendTestCase) RunTests(backendSaveTagsInMeta bool) {
```

Comment:
because in redis we are putting the tags to additional keys (SADD and SMEMBERS). if we would put the keys additionaly to the value of the key, we would have unessacary network-traffic and memory usage.